lib/src/logging.rs

```diff
@@ -381,7 +381,12 @@ macro_rules! log {
         }
       }
     });
-    ($lvl:expr, $($arg:tt)+) => (log!(target: module_path!(), $lvl, $($arg)+));
+    ($lvl:expr, $($arg:tt)+) => ({
+      // If we build in release mode and the log level is <= to Info (Debug and Trace for now), don't print the log
+      if cfg!(debug_assertions) || (! cfg!(debug_assertions) && $lvl <= $crate::logging::LogLevel::Info) {
```

Comment:
Fixed, since you said "like slog" well.. :D (I think, if I understand slog code right, that they use runtime checks)